electrum/daemon.py

```diff
@@ -708,7 +709,8 @@ def _check_password_for_directory(self, *, old_password, new_password=None, wall
                 wallet.update_password(old_password_real, new_password, encrypt_storage=True)
         can_be_unified = failed == []
         is_unified = can_be_unified and is_unified
-        return can_be_unified, is_unified
+        could_be_unlocked = len(wallet_filenames) - len(failed)
+        return can_be_unified, is_unified, could_be_unlocked
```

Comment:
`could_be_unlocked` vs `can_be_unlocked` vs `count_wallets_unlocked`
You have three different names for this single thing already in the PR that introduces it :)

Also, if the name starts with `can` or `could` or `is`, etc, by convention it is usually a boolean, which is not the case here.

I suggest changing the name to e.g. `num_can_unlock`, but in any case, be consistent.

```suggestion
        num_can_unlock = len(wallet_filenames) - len(failed)
        return can_be_unified, is_unified, num_can_unlock
```

Reply:
Yeah right that got out of hand, changed it in https://github.com/spesmilo/electrum/pull/10345/commits/ea35fe2452e1d00f60f78dfbd7c818b7a047fb6d